docs/specifications/cva6_requirement_specification.adoc

```diff
@@ -0,0 +1,391 @@
+
+The reader’s attention is drawn to the difference between an optional feature (“...shall support as an option...”) and a desired goal (“...should support...”, “....should reduce latency...”).
+
+These are not in the scope of this specification:
+
+* SW layers, such as compiler and OSes (that can however be part of the OpenHW Group CVA6 project);
+* SW emulation of RISC-V optional extensions ( feasible but the scope of the IP is the core hardware);
+* Other features included in the testbench (main memory, firmware, interconnect…), the coverage of which will not be measured;
+* The vector coprocessor (CV-VEC) that is planned to interface with CV64A6.
+
+=== Verified configurations
+
+It is not possible to verify all 2^n^ combinations of parameters. Below is the list of configurations that will undergo verification in the project and their main parameters. The full list of parameters for each configuration will be detailed in the users’ guide.
+
+|===
+|Config#|Target|32/64|FPU|MMU|L1 D$|L1 I$
+
+|cv32a6_imacf_sv32|FPGA|CV32A6|F|Sv32|32 kB|16 kB
+|cv32a6_imac_sv32|FPGA|CV32A6|None|Sv32|32 kB|16 kB
+|cv64a6_imacfd_sv39|ASIC|CV64A6|FD|Sv39|16 kB|16 kB
+|cv32a6_imac_sv0|ASIC|CV32A6|None|None (Bare mode)|None|4 kB
+|===
+
+== References
+
+=== Applicable specifications
+
+To ease the reading, the reference to these specifications can be implicit in the requirements below. For the sake of precision, the requirements identify the versions of RISC-V extensions from these specifications.
+
+[RVunpriv]   “The RISC-V Instruction Set Manual, Volume I: User-Level ISA, Document Version 20191213”, Editors Andrew Waterman and Krste Asanović, RISC-V Foundation, December 13, 2019.
+
+[RVpriv]  “The RISC-V Instruction Set Manual, Volume II: Privileged Architecture, Document Version  20211203”,  Editors  Andrew  Waterman, Krste  Asanović and John Hauser,  RISC-V Foundation, December 4, 2021.
+
+[RVdbg] “RISC-V External Debug Support, Document Version 0.13.2”, Editors Tim Newsome and Megan Wachs, RISC-V Foundation, March 22, 2019.
+
+[RVcompat] “RISC-V Architectural Compatibility Test Framework”, https://github.com/riscv-non-isa/riscv-arch-test[https://github.com/riscv-non-isa/riscv-arch-test]. +
+
+[AXI] AXI Specification, https://developer.arm.com/documentation/ihi0022/hc[https://developer.arm.com/documentation/ihi0022/hc]. +
+
+[CV-X-IF] Placeholder for the CV-X-IF coprocessor interface currently prepared at OpenHW Group; current version in https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/[https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/]. +
+
+[OpenPiton] “OpenPiton Microarchitecture Specification”, Princeton University, https://parallel.princeton.edu/openpiton/docs/micro_arch.pdf[https://parallel.princeton.edu/openpiton/docs/micro_arch.pdf]. +
+
+== Reference specifications
```

Comment:
What is the difference between Applicable and Reference specifications? Maybe just Specifications?

Reply:
This is a distinction I have been used to make (a bit the same way I make a clear distinction between requirement and informative text).
Applicable = These documents partly or wholly apply to the requirements contained in this document.
Reference = This is for information and does not directly apply to requirements.
BTW, I have renamed "Reference specifications" into "Reference documents".